Cauldron/Assets/Scripts/UI/ParallaxObject.cs

```diff
@@ -13,10 +14,11 @@ private void Start()
 
     private void LateUpdate()
     {
-        float x, y;
-        x = (Input.mousePosition.x - Screen.width / 2) * parallaxPower / Screen.width;
-        y = (Input.mousePosition.y - Screen.height / 2) * parallaxPower * PARALLAX_VERTICAL_POWER_MODIFIER / Screen.height;
-        
-        transform.position = StartPosition + new Vector3(x, y, 0);
+        float x = (Input.mousePosition.x - Screen.width / 2) * parallaxPower / Screen.width;
+        float y = (Input.mousePosition.y - Screen.height / 2) * parallaxPower * PARALLAX_VERTICAL_POWER_MODIFIER / Screen.height;
+
+        transform.position =  Vector2.Lerp(transform.position,
+                                                          StartPosition + new Vector2(x, y),
+                                                          parallaxSpeed * Time.unscaledDeltaTime);
```

Comment:
It would be cool to add a speed modifier for the distance between current point and target point, so that the movement would rush if the distance is large and gradually come to a stop when closing on target. If you understand what I mean)

Reply:
https://funkymonkey.studio/ We have a nice parallax on a corporate website to compare with))